### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Raulkumar**
updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,10 +107,10 @@ The recommended way of using `@smogon/calc` in a web browser is to **configure y
 ([Webpack][6], [Rollup][7], [Parcel][8], etc) to minimize it and package it with the rest of your
 application. If you do not use a bundler, a convenience `production.min.js` is included in the
 package. You simply need to depend on `./node_modules/@smogon/calc/production.min.js` in a `script`
-tag (which is what the unpkg shortcut above is doing), after which **`calc` will be
+tag (which is what the unpackage shortcut above is doing), after which **`calc` will be
```
Comment (scheibo) [inline README.md:110]: please revert, its unpkg, not "unpackage"

File: README.md [hunk 2]
```diff
@@ -111,10 +107,10 @@ The recommended way of using `@smogon/calc` in a web browser is to **configure y
...
 accessible as a global.** You must also have a `Generations` implementation provided, you can either
 depend on the calculator's data layer by depending on
-`./node_modules/@smogon/calc/data/production.min.js` (or `@smogon/calc/data` via unpkg), or you can
+`./node_modules/@smogon/calc/data/production.min.js` (or `@smogon/calc/data` via unpackage), or you can
```
Comment (scheibo) [inline README.md:113]: ditto

File: README.md [hunk 3]
```diff
@@ -168,15 +164,15 @@ how to programmatically update sets.
 
 ## Credits
 
-This project was originally created by Honko and is primarly maintained by Austin and pre.
+This project was created by Honko and is primarily maintained by Austin and pre.
```
Comment (scheibo) [inline README.md:167]: While this is being updated, can you please do me a favor and remove "and pre"? Thanks!

File: README.md [hunk 4]
```diff
@@ -32,26 +32,22 @@ a convenient way to get started, simply depend on a transpiled and minified vers
...
-- attacker and defender `Pokemon` (note: only relevant attributes are required, everything else
-  should have sensible defaults). The `Pokemon` constructor also requires a `Generation` to provide
-  the Pokémon's data for the generation.
-- the `Move` being used by the attacker (which also requires a `Generation` argument to scope the
-  move data to the particular generation).
+methods require:
+
+- a `Generation` which contains information about which damage formula mechanics to apply and where all of the data about the generation can be found.
+- attacker and defender `Pokemon` (note: only relevant attributes are required, everything else should have sensible defaults). The `Pokemon` constructor also requires a `Generation` to provide the Pokémon's data for the generation.
+- the `Move` being used by the attacker (which also requires a `Generation` argument to scope the move data to the particular generation).
```
Comment (KrisXV) [inline README.md:46]:
```suggestion
- a `Generation` that contains information about which damage formula mechanics to apply and where
  all of the data about the generation can be found.
- attacker and defender `Pokemon` (note: only relevant attributes are required, everything else
  should have sensible defaults). The `Pokemon` constructor also requires a `Generation` to provide
  the Pokémon's data for the generation.
- the `Move` being used by the attacker (which also requires a `Generation` argument to scope the
  move data to the particular generation).
```

File: README.md [hunk 5]
```diff
@@ -32,26 +32,22 @@ a convenient way to get started, simply depend on a transpiled and minified vers
...
-
-- a `Generation` which contains information about which damage formula mechanics to apply and where
-  all of data about the generation can be found.
-- attacker and defender `Pokemon` (note: only relevant attributes are required, everything else
-  should have sensible defaults). The `Pokemon` constructor also requires a `Generation` to provide
-  the Pokémon's data for the generation.
-- the `Move` being used by the attacker (which also requires a `Generation` argument to scope the
-  move data to the particular generation).
+methods require:
+
```
Comment (scheibo) [inline README.md:43]: Please revert, the line wrapping is intentional

**Review by scheibo — COMMENTED** [review]
This LGTM pending nits, thanks for fixing my typos!

File: README.md [hunk 6]
```diff
@@ -74,22 +70,22 @@ const result = calculate(
...
+`@smogon/calc` comes packaged with all of the data required for damage calculation - by default, it
 exposes this via its `Generations` object from `@smogon/calc/data`. As a shortcut, the `Generation`
 argument required by `calculate`, `Pokemon`, `Move` can instead simply be the generation *number*
 (eg. `5`), and it will handle getting that generations `Generation` object behind the scenes from
 the data layer it ships with.
 
 **The data in `calc/data` must be kept in sync with Pokémon Showdown. If there is an issue with the
-calc's data, please first fix it in the simulator first.** In general, you should probably not be
-making manual edits to any of the data files, and in the future they are likely to be generated
+calc's data, please first fix it in the simulator.** In general, you should probably not be
```
Comment (KrisXV) [inline README.md:80]:
```suggestion
calc's data, please fix it in the simulator first.** In general, you should probably not be
```

File: README.md [hunk 7]
```diff
@@ -149,7 +145,7 @@ $ open dist/index.html # open works on macOS, simply double-clicking the file on
 **If you make changes to anything in `calc/`, you must run `node build` from the top level to
 compile the files and copy them into `dist/` again. If you make changes to the HTML or JavaScript in
 `src/`you must run `node build view` before the changes will become visible in your browser**
-(`node build` also works, but is slower as it also will compile `calc/` as well, which is
+(`node build` also works but is slower as it will compile `calc/` as well, which is
```
Comment (KrisXV) [inline README.md:148]:
```suggestion
(`node build` also works, but it is slower, as it will compile `calc/` as well, which is
```

**Review by KrisXV — DISMISSED** [review]
Please do the changes I suggested and @scheibo requested.

File: README.md [hunk 8]
```diff
@@ -168,15 +164,15 @@ how to programmatically update sets.
...
 
 - Gens 1-6 were originally implemented by Honko.
 - The Omega Ruby / Alpha Sapphire update was done by gamut-was-taken and Austin.
 - The Gen 7 update was done by Austin.
 - The Gen 8 update was done by Austin.
 - Some CSS styling was contributed by Zarel to match the Pokémon Showdown! theme.
 
-A number of other contributors have added features or contributed bug fixes, please see the
+Many other contributors have added features or contributed bug fixes, please see the
 [full list of contributors](https://github.com/smogon/damage-cDalc/graphs/contributors).
```
Comment (AnnikaCodes) [inline README.md:176]:
```suggestion
[full list of contributors](https://github.com/smogon/damage-calc/graphs/contributors).
```

**Pull request merged**